plugins/treo-websql/indexeddb-shim.js

```diff
@@ -1588,6 +1583,14 @@ var cleanInterface = false;
             var req = new idbModules.IDBOpenRequest();
             var calledDbCreateError = false;
 
+            sysdb.transaction(function(tx){
+                tx.executeSql("CREATE TABLE IF NOT EXISTS dbVersions (name VARCHAR(255), version INT);", [], function(){
```

Comment:
Do you need the additional anonymous function? Could you just `[], updateVersions);`?